src/applications/personalization/preferences/components/PreferenceItem.jsx

```diff
@@ -52,57 +52,81 @@ const FAQList = ({ faqs }) => (
   </div>
 );
 
-export default function PreferenceItem({
-  handleViewToggle,
-  handleRemove,
-  isRemoving,
-  benefit,
-}) {
-  const { title, introduction, code, cta, faqs } = benefit;
+export default class PreferenceItem extends React.Component {
+  constructor(props) {
+    super(props);
 
-  if (isRemoving) {
+    this.removeBtnRef = React.createRef();
+    this.alertHeaderRef = React.createRef();
```

Comment:
More React 16 🔥

Reply:
Is this a deal-breaker, @ncksllvn?
Erik & I discussed these refs and decided not to replace these refs w/ the new Hooks (which I'm just learning about now).

Reply:
I read that the `createRef` is part of React 16 so that’s all I meant! I’m really enjoying seeing it in action in your work.

Reply:
Oh, I thought you wanted to flame it!  ROFL!